Parking/ParkingAccess/doc/spec.md

```diff
@@ -14,7 +14,7 @@ Represents an access point to a parking site, normally an offstreet parking.
     + Attribute type: `geo:json`.
     + Normative References: [https://tools.ietf.org/html/rfc7946](https://tools.ietf.org/html/rfc7946)
     + Mandatory
-    
+
```

Comment:
please do not modify data models without substantive changes. please stick to the PR

Reply:
Sorry, this change was in for the docs to pass the checks.